spec/syntax.md

```diff
@@ -369,6 +369,8 @@ satisfied:
 - At least one _variant_ MUST exist whose _keys_ are all equal to the "catch-all" key `*`.
 - Each _selector_ MUST have an _annotation_,
   or contain a _variable_ that directly or indirectly references a _declaration_ with an _annotation_.
+- Every _variant_ MUST NOT use the same set of _keys_ in the same order as another _variant_.
```

Comment:
I suggested "list" above and would suggest it here as well. Sets, as you know, can be unordered, while lists generally are ordered.

```suggestion
Each _variant_ MUST use a list of _keys_ that is unique from that
of all other _variants_ in the _message_.
```

Reply:
That makes sense; there's another "set" mention that ought to be enlisted as well. Fixed your indents:
```suggestion
- Each _variant_ MUST use a list of _keys_ that is unique from that
  of all other _variants_ in the _message_.
```